scripts/manage-isos.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -u
+
+usage()
+{
+    echo "----------------------------------------------------------------------------------------"
+    echo "- Usage: $0 [version XCP-ng] [Repository] [Server PXE] [Path on the server PXE] [Server SR ISO] [Path on the server of the SR ISO] [Path on the pXE for th installer]"
```

Comment:
"PXE server", "ISO SR"

Reply:
"for th installer" -> "for the installer"